src/libraries/Microsoft.Win32.Registry.AccessControl/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Registry.AccessControl
+
+This assembly provides APIs for managing access control lists for Microsoft.Win32.RegistryKey.
```

Comment:
We should match the message we have in the package: https://github.com/dotnet/runtime/blob/28cf1719704d33a677990cf487af5719b6389f2a/src/libraries/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj#L6

It is also the message we publish in the NuGet Readme: https://www.nuget.org/packages/Microsoft.Win32.Registry.AccessControl#readme-body-tab

```suggestion
This assembly provides APIs for managing access and audit control lists for Microsoft.Win32.RegistryKey.
```